source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet

```diff
@@ -76,4 +76,21 @@ local pastProxyTitle = {
       when: [rules.isProxy, rules.lastMainJob],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
+        group: 'school-group',
+        when: [{
+          id: 'in-education-answer',
+          condition: 'equals',
+          value: 'Yes',
+        }],
+      },
+    },
+    {
+      goto: {
+        block: 'work-location-type',
+      },
+    },
+  ],
```

Comment:
Did a journey where I answered `Studying` and past job, but from hours worked I went to `work-location-type` but card says it should go to `study-location-type` if `in-education` is `yes` or `employment-type` is `studying`.

Reply:
I tried the same journey and it worked in my local environment?!

Reply:
Have fixed this, added an extra clause for employment-type